dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -130,7 +130,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     bool PushNotificationManager::IsSupported()
     {
         // Only scenarios that use the Background Infrastructure component of the OS support Push in the SelfContained case
```

Comment:
Update the comment to:
Packaged App scenario always works
[4:25 PM] Pavan Hullumane
Unpackaged App scenario does not

[4:25 PM] Pavan Hullumane
Unpackaged does not when it is self-contained or it is elevated